index.js

```diff
@@ -9,65 +9,114 @@ const newMember = require("./data/new-member");
  * @param {Array[]} member -
  * @return {number} The sum of the numbers in an array
  */
-function newMemberArrayToObject(member) {}
+function newMemberArrayToObject(member) {
+  return _.fromPairs(member);
+}
 
 /**
  * Return an array of objects that grouped by instructors from the classes array of objects
  * @param {Object[]} collection - an array of yoga class objects
  * @return {Object[]} - the reshaped collection where the classes are grouped by instructor name
  */
-function groupClassByInstructor(collection) {}
+function groupClassByInstructor(collection) {
+  return _.groupBy(collection, "instructor");
+}
 
 /**
  * Remove the age key from the members array of object
  * @param {Object} collection - an array of member objects
  * @return {number} The array of member objects, each one without the age field
  */
-function omitAgeFromMembers(collection) {}
+function omitAgeFromMembers(collection) {
+  return _.map(collection, member => _.omit(member, `age`));
+}
 
 /**
  * Return the count of the number of classes a particular instructor teaches
  * @param {Object[]} collection - An array of yoga class objects
  * @param {String} instructorName - The name of the instructor
  * @return {number} The sum of the numbers in an array
  */
-function countClassesByInstructor(collection, instructor) {}
+function countClassesByInstructor(collection, instructorName) {
+  const classByInstructor = _.filter(collection, { instructor: instructorName });
+  
+  if (classByInstructor.length > 0) {
+    return classByInstructor.length;
+  } else {
+    return "There is no instructor by that name.";
+  }
+}
 
 /**
  * Remove inactive members from the members array
  * @param {Object} collection - an array of member objects
  * @return {number} The array of member objects with only active members
  */
-function removeInactiveMembers(collection) {}
+function removeInactiveMembers(collection) {
+  return _.filter(collection, ["currentMember", true]);
+}
 
 /**
  * Get a list of unique class titles and their price
  * @param {Object} collection - an array of yoga class objects
  * @return {number} An array of objects that have a unique title and a price
  */
-function getUniqueClasses(collection) {}
+function getUniqueClasses(collection) {
+  return _.uniqBy(collection, 'title').map(item => ({
+    title: item.title,
+    priceInCents: item.priceInCents,
+  }));
+}
 
 /**
  * Get a list of classes organized by title, then by level.
  * The titles should be in ascending order, the levels should be in descending order
  * @param {Object} collection - an array of yoga class objects
  * @return {number} An array of objects that are organized by title then by level. The array should only have the title, instructor, and level fields
  */
-function orderClassesByTitleAndLevel(collection) {}
+function orderClassesByTitleAndLevel(collection) {
+  const groupByTitle = _.groupBy(collection, 'title');
```

Comment:
Note that you could use the orderBy method to shorten this down. It would take multiple values for the sort order and gives you back the classes based on the fields you wanted to sort by. That way, you don't have to create separate groups and loop over those groups.